build.gradle

```diff
@@ -750,3 +750,29 @@ class DockerPushTask extends Exec {
         }
     }
 }
+
+apply from: "$rootDir/gradle/mkdocs.gradle"
+mkdocs {
```

Comment:
Can we move mkdocs and remotes configuration and the deploy task to mkdocs.gradle?

Reply:
done.